### ISSUE TITLE
Possible paper to  noctua model (cytoskeleton, chromosome segregation)

### ISSUE THREAD

**Issue by manulera** [pombase/curation#3267]
Following curation of PMID:24790093 I think this could be a good paper to try to automate the generation of a Noctua model. We can discuss. See the image below (maybe you even mentioned this paper for Noctua already?):

![Screenshot 2022-06-01 at 18 57 18](https://user-images.githubusercontent.com/22526102/171471383-b7b1105e-9287-48dd-b125-92ed67f1bb40.png)

**Comment by dexink** [comment]
@ValWood  I would love a semi-supervised SOM model for producing these forms of curator-QC-dependent processes
Prompts for unsupervised preliminary models to be checked, approved, and then submitted for finalization and algorithm update would be optimal (both highly flexible and adequately cautious).

**Comment by ValWood** [comment]
This is a nice example. 
Where is the regulation?
Cut15 is constitutive (what does Paul call this), the regulation is via cdk phosphorylation of alp7

**Comment by manulera** [comment]
Yes, alp7 phosphorylation by CDK during mitosis increases its interaction with Cut15, leading to the accumulation of alp7 in the nucleus. Perhaps it's not the best example, as it's not an on/off switch, but a displacement of the equilibrium towards having more alp7-alp14 in the nucleus.
